packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -105,14 +114,31 @@ export class Tooltip {
           if (computeResponse.middlewareData.arrow) {
             let { x, y } = computeResponse.middlewareData.arrow;
 
-            if (computeResponse.placement === 'bottom') {
-              y = -4;
+            const toCSSValue = (value: number) =>
+              value != null ? `${value}px` : '';
+            let arrowPosition = {};
+
+            if (computeResponse.placement.startsWith('top')) {
```

Comment:
Just a minor thing but would it not be better to connect these conditions with else?

Reply:
Its refactored but still without `else` branch :)